project/src/pages/welcome-page/welcome-page.tsx

```diff
@@ -113,11 +116,9 @@ function WelcomePage({offerCount}: WelcomePageProps): JSX.Element {
                   <li className="places__option" tabIndex={0}>Top rated first</li>
                 </ul>
               </form>
-              <div className="cities__places-list places__list tabs__content">
-                {
-                  Array(offerCount).fill(<OfferCard/>)
-                }
-              </div>
+              {
+                <OffersList offers={offers} offerCount={offerCount}/>
+              }
```

Comment:
когда компонент подключаешь фигурные скобки не нужны 
```suggestion
              
                <OffersList offers={offers} offerCount={offerCount}/>
              
```